src/ui/textbox/textbox.android.ts

```diff
@@ -82,7 +82,7 @@ export default class TextBoxAndroid<TEvent extends string = TextBoxEvents, TNati
   private _isPassword: boolean;
   private _keyboardType: KeyboardType;
   private _actionKeyType: ActionKeyType;
-  private _maxLength: number;
+  private _maxLength: number | undefined;
```

Comment:
I believe we can remove the undefined type here. If removing cause an error on getter, check the ios implementation please. @furkanarabaci  said that, instead of using "undefined" we can just return zero